draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -164,10 +165,16 @@ after we submit a new draft to ikev2-beyond64k-limit. -->
 			the IKE_AUTH exchange, we perform one or more IKE_INTERMEDIATE exchanges,
 			each of which contains an additional key exchange.  As the IKE_INTERMEDIATE
 			exchange is encrypted, the IKE fragmentation protocol <xref target="RFC7383" />
-			can be used. The IKE SK_* values are updated after each exchange,
+			can be used. The IKE SK_* values are updated after each exchange as described in
+            <xref target="additional_ke"/>,
 			and so the final IKE SA keys depend on all the key exchanges,
 			hence they are secure if any of the key exchanges are secure.</t>
 
+            <t>While this extension is primarily aimed for IKE SAs due to the
+            potential fragmentation issue discussed above, it also applies to
+            CREATE_CHILD_SA exchanges for creating/rekeying of Child SAs and 
+            rekeying of IKE SAs.</t>
+
```

Comment:
Shouldn't here be reference to "create_child_sa_exchange" section? I notice you've added the anchor there, but failed to find where it is used.

Reply:
Yep, I forgot about this, will fix in the next commit.